src/OHD.ts

```diff
@@ -384,6 +384,12 @@ export default class OHD {
       };
     });
   }
+  /**
+   * Add `admin_name` to admins list
+   */
+  public addAdminName(player_name = 'whatAboutBob'): Promise<void> {
+    return this.send(`admin add ${player_name}`) as Promise<void>;
```

Comment:
Usernames **MUST** be encapsulated in quotes, or if the name has a space the command will fail.

```diff
- admin add Snow E
+ admin add "Snow E"
```